requirements/base.txt

```diff
@@ -66,62 +67,63 @@ drf-yasg==1.20.0
     # via edx-api-doc-tools
 edx-api-doc-tools==1.5.0
     # via -r requirements/base.in
-edx-auth-backends==4.0.0
+edx-auth-backends==4.0.1
     # via -r requirements/base.in
 edx-django-release-util==1.1.0
     # via -r requirements/base.in
-edx-django-utils==4.4.0
+edx-django-utils==4.4.1
     # via -r requirements/base.in
-idna==3.2
+idna==3.3
     # via requests
 inflection==0.5.1
     # via drf-yasg
 itypes==1.2.0
     # via coreapi
-jinja2==3.0.2
+jinja2==3.0.3
     # via coreschema
 markupsafe==2.0.1
     # via jinja2
-mysqlclient==2.0.3
+mysqlclient==2.1.0
     # via -r requirements/base.in
-newrelic==7.0.0.166
+newrelic==7.2.4.171
     # via edx-django-utils
 oauthlib==3.1.1
     # via
     #   requests-oauthlib
     #   social-auth-core
-packaging==21.0
+packaging==21.3
     # via drf-yasg
-pbr==5.6.0
+pbr==5.8.0
     # via stevedore
-psutil==5.8.0
+psutil==5.9.0
     # via edx-django-utils
 pyblake2==1.1.2
     # via -r requirements/base.in
-pycparser==2.20
+pycparser==2.21
     # via cffi
-pyjwt[crypto]==2.1.0
+pyjwt[crypto]==2.3.0
     # via
     #   edx-auth-backends
     #   social-auth-core
-pyparsing==2.4.7
+pyparsing==3.0.6
     # via packaging
 python3-openid==3.2.0
     # via social-auth-core
 pytz==2021.3
     # via
     #   -r requirements/base.in
     #   django
-pyyaml==5.4.1
+    #   djangorestframework
+pyyaml==6.0
```

Comment:
Hi @doctoryes CC @saksham115 @gabor-boras

We're getting an error when running migrations for the latest master blockstore. I haven't confirmed for sure, but the error looks like it's related to this change.

```
# /edx/app/blockstore/venvs/blockstore/bin/python manage.py migrate --no-input
  "Traceback (most recent call last):",
        "  File \"manage.py\", line 15, in <module>",
        "    execute_from_command_line(sys.argv)",
        "  File \"/edx/app/blockstore/venvs/blockstore/lib/python3.8/site-packages/django/core/management/__init__.py\", line 419, in execute_from_command_line",
        "    utility.execute()",
        "  File \"/edx/app/blockstore/venvs/blockstore/lib/python3.8/site-packages/django/core/management/__init__.py\", line 363, in execute",
        "    settings.INSTALLED_APPS",
        "  File \"/edx/app/blockstore/venvs/blockstore/lib/python3.8/site-packages/django/conf/__init__.py\", line 82, in __getattr__",
        "    self._setup(name)",
        "  File \"/edx/app/blockstore/venvs/blockstore/lib/python3.8/site-packages/django/conf/__init__.py\", line 69, in _setup",
        "    self._wrapped = Settings(settings_module)",
        "  File \"/edx/app/blockstore/venvs/blockstore/lib/python3.8/site-packages/django/conf/__init__.py\", line 170, in __init__",
        "    mod = importlib.import_module(self.SETTINGS_MODULE)",
        "  File \"/usr/lib/python3.8/importlib/__init__.py\", line 127, in import_module",
        "    return _bootstrap._gcd_import(name[level:], package, level)",
        "  File \"<frozen importlib._bootstrap>\", line 1014, in _gcd_import",
        "  File \"<frozen importlib._bootstrap>\", line 991, in _find_and_load",
        "  File \"<frozen importlib._bootstrap>\", line 975, in _find_and_load_unlocked",
        "  File \"<frozen importlib._bootstrap>\", line 671, in _load_unlocked",
        "  File \"<frozen importlib._bootstrap_external>\", line 848, in exec_module",
        "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed",
        "  File \"/edx/app/blockstore/blockstore/blockstore/settings/production.py\", line 23, in <module>",
        "    config_from_yaml = yaml.load(f)",
        "TypeError: load() missing 1 required positional argument: 'Loader'",
        "make: *** [Makefile:79: migrate] Error 1"
```

Could you have a look?

Reply:
@pomegranited I looked into it and the error is being caused by this deprecation, which has become a full-blown error in pyyaml 6.0+:
https://github.com/yaml/pyyaml/wiki/PyYAML-yaml.load(input)-Deprecation

The fix to use `yaml.safe_load()` instead of `yaml.load()`. I'll make that change now.

Reply:
Fix will be in this PR: https://github.com/openedx/blockstore/pull/152